config/config.yml

```diff
@@ -52,6 +52,7 @@ defaults: &defaults
       format: 'application/xml'
   metadata:
     url: 'https://purl.dlib.indiana.edu/iudl/iucat/%s'
+  purl_redirect_url: 'http://server1.variations2.indiana.edu/cgi-bin/access?%s'
```

Comment:
@aploshay Adam I didn't know if notifications work when changes are requested on a review. No hurry, just didn't know if you saw the comment.